templates/buy-me-a-coffee-app/src/components/Logo.tsx

```diff
@@ -0,0 +1,98 @@
+type LogoProps = {
+  color?: 'white' | 'black';
+  width?: string;
+  height?: string;
+};
+
+export function Logo({ width = 'auto', height = '40' }: LogoProps) {
```

Comment:
Is this a standard Logo used for this demo app? I wonder if we should customize it and have something more our own.